public/app/features/plugins/admin/api.ts

```diff
@@ -59,6 +64,15 @@ async function getPlugin(slug: string): Promise<PluginDetails> {
   };
 }
 
+async function getPluginErrors(): Promise<PluginError[]> {
+  try {
+    const pluginErrors = await getBackendSrv().get(`${API_ROOT}/errors`);
+    return pluginErrors;
```

Comment:
**Nit:** I think we can just return with the `await` and still being able to catch the errors

```suggestion
    return await getBackendSrv().get(`${API_ROOT}/errors`);;
```

Reply:
Hehe, not sure why I did that but great call!